### ISSUE TITLE
Change Request for SubPanel

### ISSUE THREAD

**Issue by CuriaPLUS** [Lambda-IT/parlamentsdienste-components#257]
Hey
We need some changes for the Subpanel in order of our UX-Review. 

- If a Subpanel is open, there should be no border at the bottom (closed panel should have border-top and border-bottom as it is)
- Nested Subpanels should be intended

Here a Screenshot for better understanding :)

<img width="1100" alt="image" src="https://github.com/Lambda-IT/parlamentsdienste-components/assets/77280472/e10413ca-7658-4a22-b36a-63a8f7683ed4">

Best Regards

**Comment by tom-wenger** [comment]
Hello
If I understand you correctly, the screenshot shows how it should be?
Should these changes apply to all subpanels by default or should it be possible to control it via a property?
👍

**Comment by d3orn** [comment]
- If a Subpanel is open, there should be no border at the bottom (closed panel should have border-top and border-bottom as it is) -> Should always be the case, if a Subpanel is opened the bottom line can be removed
- Nested Subpanels should be intended -> I would make this a prop but the default should be indented with the option to go back to the current version

**Comment by tom-wenger** [comment]
I made a release today [v2.6.13](https://github.com/Lambda-IT/parlamentsdienste-components/releases/tag/v2.6.13) and i removed the border bottom on opened subpanels. 
Let me know if that's what you need.

**Comment by d3orn** [comment]
Some minor feedback:

- Hover still shows the bottom line (top hover is fine)
- Nested Subpanels should to be indented: I guess we could do this with --pd-panel-content-padding as we do not have a lot of nested subpanels. But if you see a easy way to intend as soon as subpanels are nested that would be nice as well

**Comment by tom-wenger** [comment]
I implemented this in the latest release. attention: i removed --pdd-panel-margin-bottom css property because it affected the margin of the subpanels and the main panel as well and that seemed very confusing.
I added --pd-panel-subpanel-margin. Only effectiv if panel is a subpanel. Can be used to separate and indent subpanels. If nothing else is set, it has a default value of 0 0 1rem 1rem to indent the subpanels slightly.
